public/content/restaking/index.md

```diff
@@ -0,0 +1,177 @@
+
+## What are the risks of restaking? {#what-are-the-risks-of-restaking}
+
+To truly appreciate the impact of Ethereum payments, it's worth comparing them to traditional fiat currencies:
+
+| **Risks**                     | **Description**                                                                                                                                                | **Risk level** |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| **Penalties (or “slashing”)** | Like ETH staking, if restakers/operators go offline, censor messages or try to corrupt the network, their stake can be slashed (burned) partially or entirely. | LOW            |
+| **Centralization**            | If few operators dominate most of the restaking they could have a great influence on restakers, AVSs and even restaking platforms.                             | LOW            |
+| **Chain reactions**           | If a restaker gets slashed while securing multiple AVSs, this could lower the security for the other AVSs, making them vulnerable.                             | LOW/MEDIUM     |
+| **Immediate access to funds** | There is a wait time (or “unbonding period”) for withdrawing restaked ETH so you may not always have access immediately.                                       | LOW/MEDIUM     |
+
+<br/>
+
+<InfoBanner shouldSpaceBetween emoji=":bulb:">
+  <div>
+    <p><strong>The Ethereum co-founder is typing…</strong></p>
+    <p>
+      Vitalik, the co-founder of Ethereum, warned about the potential risks of restaking in a 2021 blog post called <a href = "https://vitalik.eth.limo/general/2023/05/21/dont_overload.html"> Don't Overload Consensus. </a>
+    </p>
+  </div>
+</InfoBanner>
+
+## How to get started with restaking? {#how-to-get-started-with-restaking}
+
+| 🫡 Beginners                                                    | 🤓 Advanced Users                                                                     |
+| --------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| 1. Stake ETH on platforms like Lido or Rocket Pool to get LSTs. | 1. Stake your ETH as a validator on Ethereum.                                         |
+| 2. Use those LSTs to start restaking on EigenLayer.             | 2. Compare restaking services like EigenLayer, Symbiotic and others.                  |
+|                                                                 | 3. Follow the instructions to connect your validator to the restaking smart contract. |
+
+<br/>
+
+<InfoBanner shouldSpaceBetween emoji=":eyes:">
+  <div><strong>Ethereum Staking :</strong> How does it work?</div>
+  <ButtonLink href="/staking/">
+    Learn More
+  </ButtonLink>
+</InfoBanner>
+
+## Advanced {#advanced}
+
+<YouTube id="-V-fG4J1N_M" />
```

Comment:
We should have at least some content here 🙏

Reply:
maybe we can remove this so that there is less focus on eigen